Geometry/TrackerGeometryBuilder/BuildFile.xml

```diff
@@ -7,6 +7,10 @@
 <use name="Geometry/CommonDetUnit"/>
 <use name="Geometry/CommonTopologies"/>
 <use name="Geometry/TrackerNumberingBuilder"/>
+<use name="CondFormats/GeometryObjects"/>
```

Comment:
@davidlange6 , looks like this is causing cyclic deps in cxxmodules IBs due to following dep chain
CondFormats/GeometryObjects -> Geometry/VeryForwardGeometryBuilder -> Geometry/TrackerNumberingBuilder -> CondFormats/GeometryObjects

module build failed to compile any thing,